src/all/_all.scss

```diff
@@ -1,5 +1,9 @@
+// optional setting of global styles for <a> and <p>
+$govuk-global-styles: true !default;
+
 // Settings
 @import "../globals/common";
+@import "../globals/optional/global-styles";
```

Comment:
Given the context `optional/` may be the right name.
My view of this was that we'd have a `/core` in which we place the default style that you can choose not to use.

Reply:
was considering putting it in core first. i have no issue with it being in core tbh

Reply:
I quite like `optional/` as this clearly differentiates those styles from the rest. But not a blocker.